src/jlapi.c

```diff
@@ -310,6 +310,13 @@ DLLEXPORT jl_taggedvalue_t *jl_astaggedvalue(jl_value_t *v)
     return jl_astaggedvalue__MACRO(v);
 }
 
+// Create function versions of some useful macros
+#undef jl_valueof
+DLLEXPORT jl_value_t *jl_valueof(jl_taggedvalue_t *v)
+{
+    return jl_valueof__MACRO(v);
+}
```

Comment:
> So are you sure this is fine with all pre-processors now?

Yes. With further research, I realized this works because it hides the essential parentheses from the function call macro, not just some random implementation-specific behavior.